test_data/config_2013_minneapolis_park.json

```diff
@@ -53,7 +53,7 @@
     "maxRankingsAllowed": 3,
     "maxSkippedRanksAllowed": 2,
     "numberOfWinners": 3,
-    "decimalPlacesForVoteArithmetic": 5,
+    "decimalPlacesForVoteArithmetic": 4,
```

Comment:
Also fix for "config_2015_portland_mayor_multi.json" and "config_test_continue_tabulation.json"?

Reply:
This is only relevant for multi-seat contests, and I think it varies by jurisdiction. We can update other configs if/when we determine it's appropriate.